driver/ni/digital_read.cpp

```diff
@@ -7,144 +7,173 @@
 // License, use of this software will be governed by the Apache License, Version 2.0,
 // included in the file licenses/APL.txt.
 
-//
-// Created by Emiliano Bonilla on 1/3/24.
-//
-
-#include "driver/ni/ni.h"
-#include "nlohmann/json.hpp"
-#include "client/cpp/telem/telem.h"
 #include <utility>
 #include <chrono>
 #include <stdio.h>
 #include <cassert>
+
+#include "client/cpp/telem/telem.h"
+#include "driver/ni/ni.h"
+
 #include "glog/logging.h"
+#include "nlohmann/json.hpp"
 
 using json = nlohmann::json;
 
-void ni::DigitalReadSource::parseChannels(config::Parser &parser){
-    LOG(INFO) << "[NI Reader] Parsing Channels for task " << this->reader_config.task_name;
+void ni::DigitalReadSource::parseChannels(config::Parser &parser) {
+    LOG(INFO) << "[NI Reader] Parsing Channels for task " << this->reader_config.
+            task_name;
     // now parse the channels
     parser.iter("channels",
-                [&](config::Parser &channel_builder){
+                [&](config::Parser &channel_builder
+        ) {
                     ni::ChannelConfig config;
-
                     // digital channel names are formatted: <device_name>/port<port_number>/line<line_number>
-                    config.name = (this->reader_config.device_name + "/port" + std::to_string(channel_builder.required<std::uint64_t>("port")) + "/line" + std::to_string(channel_builder.required<std::uint64_t>("line")));
-
+                    config.name = (this->reader_config.device_name + "/port" +
```

Comment:
This code is duplicated in `digital_write.cpp`

Reply:
changes it to make it clearer in both but think the code duplication is acceptable/doesn't create any additional clarity by taking it out